articles/rich-text-editor-reference.mdx

```diff
@@ -0,0 +1,489 @@
+
+``````````
+</comp.MultiCodeBlock>
+
+<comp.DataReference>
+  <comp.DataReferenceRow label="blockquote" type_markdown="Boolean">
+    This key enables a tool to wrap selected blocks of text in a block quote.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="bold" type_markdown="Boolean">
+    This key enables a tool to bold selected text.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), the Text editable region (`_editables.text`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="format" type_markdown="String">
+    This key enables a dropdown menu tool for structured text. Valid values include for `p`, `h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `pre`, `address`, and `div`. Include a space between each values, e.g. `format: p h1 h2`.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="italic" type_markdown="Boolean">
+    This key enables a tool to italicize selected text.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), the Text editable region (`_editables.text`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="link" type_markdown="Boolean">
+    This key enables a tool to create a hyperlink around selected text.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), the Text editable region (`_editables.text`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="strike" type_markdown="Boolean">
+    This key enables a tool to strike through selected text.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), the Text editable region (`_editables.text`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="subscript" type_markdown="Boolean">
+    This key enables a tool to subscript selected text.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), the Link editable region (`_editables.link`), the Text editable region (`_editables.text`), and Rich Text inputs (`_inputs.key_name.options`).
```

Comment:
If we want to thin down this page a little, in this case we could specify "Available for all rich text editors"

Am ok with keeping it verbose, too.

Reply:
Opting for verbose